Tests/Application/PayslipsValidatorTests.cs

```diff
@@ -0,0 +1,117 @@
+using Application.Services;
+using Core;
+using Microsoft.AspNetCore.Http;
+using Xunit;
+
+namespace Tests.Application
+{
+    public class PayslipsValidatorTests
+    {
+        private readonly IPayslipsValidator _validator = new PayslipsValidator();
+        private readonly List<Employee> _emptyEmployeeList = new();
+
+        private static async Task<IFormFile> GetFormFileAsync(string fileName, string content)
+        {
+            var stream = new MemoryStream();
+            var writer = new StreamWriter(stream);
+            await writer.WriteAsync(content);
+            await writer.FlushAsync();
+            stream.Position = 0;
+
+            return new FormFile(stream, 0, stream.Length, "file", fileName);
+        }
+
+        [Fact]
+        public async Task PayslipsValidator_WhenPayslipsListIsEmpty_ShouldThrowException()
```

Comment:
I think we shouldn't follow BDD naming in this file because we already know that payslips validator.
Can offer to rename it like: `PayslipsMustNotBeEmpty` or another naming that you prefer. Think about naming for other tests